game.rb

```diff
@@ -0,0 +1,37 @@
+require './environment.rb'
+
+class Game
+    attr_accessor :board, :depot, :control
+
+    def initialize        
+        @board = {
+            :players => {}, 
+            :control => {}, 
+            :status => nil
+        }
+        @depot = Environment::Depot.new
+        @control = Environment::Control.new
+    end
+
+
+	def new_worker(name:)
```

Comment:
Use named arguments without setting a default is not taken as a good practice, it should be better: 
`def new_worker name: nil`